### ISSUE TITLE
readline bindings - copying to primary

### ISSUE THREAD

**Issue by sdx23** [qutebrowser/qutebrowser#4298]
Readline bindings, as for example <ctrl-u> for deleting the whole line, are quite useful.
There is no native support for this in qutebrowser, so the recommended solution is to use bindings like
    <Ctrl+u>: fake-key <shift-home>;; fake-key <del>

This works, in a way. However, as a side effect the deleted string is copied to primary selection - which is both non-standard behaviour and quite unexpected. Depending on the content it might even be dangerous.

**Comment by sdx23** [comment]
For shift-Home, of course it is expectable. But in no case for Ctrl-U. This messed up something for me just a few minutes ago, so I thought it's at least worth mentioning.

**Comment by The-Compiler** [comment]
True, it's yet another reason why this should be implemented properly in JavaScript. I added a comment about it to #68.
